third_party/jitterentropy/META.yml

```diff
@@ -0,0 +1,5 @@
+name: jitterentropy-library
```

Comment:
Could we just make jitterentropy a git submodule?

Reply:
yes, we could. But we do not use it anywhere else. It would be a strictly bigger behaviour change. Probably something we should do a cross the whole library at the same time.